daemon/cluster/cluster.go

```diff
@@ -687,6 +687,19 @@ func (c *Cluster) CreateService(s types.ServiceSpec) (string, error) {
 	if err != nil {
 		return "", err
 	}
+
+	currentService, err := getService(c.getRequestContext(), c.client, serviceID)
```

Comment:
What is `serviceID` here? Why are we retrieving a service in `CreateService`? Does this compile?